engine/apps/alerts/constants.py

```diff
@@ -10,3 +13,11 @@ class ActionSource:
 TASK_DELAY_SECONDS = 1
 
 NEXT_ESCALATION_DELAY = 5
+
+
+# AlertGroup states verbal
+class AlertGroupState(str, Enum):
```

Comment:
I've yet to try this out, but with Python 3.11 there is a new class, [`StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum), might be worth giving a try